server/http/restapi/marketplace.go

```diff
@@ -0,0 +1,145 @@
+package restapi
+
+import (
+	"net/http"
+	"sort"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+type MarketplaceApp struct {
+	Installed bool                     `json:"installed"`
```

Comment:
What `Installed` means?
We have 3 states of the apps for now:
1. `registered`/`listed`
2. `enabled`
3. `disabled`

Reply:
In these terms it means `enabled`.

Are these three terms used somewhere in code or docs? I would like to use consistent naming.

Reply:
In this context, can we live with the definition registered+enabled == installed? Can one app be enabled but not listed/registered?

Reply:
@catalintomai Yes, that is ture.
Should I change app state names as `Registered`, `Installed`, `Uninstalled` to be more inline with the marketplace?

Reply:
I'd say having common naming is a definite must. Maybe sending a message to the MM Apps channel with the proposed state naming (and meaning of each) would close this discussion for all.Like: 
registered/listed - means that the app has a manifest present in s3 bucket and is present in the apps.json bundle file, etc.
enabled - ...
installed - ...
uninstalled:...

Reply:
@iomodo, let's use the naming of the marketplace then.